### PR TITLE
GCW-2394 restrict partial dispatch

### PR DESCRIPTION
**Pull request by abhinav098**
Hello team, 
This PR restricts user to partial dispatch if order is not processed.
Please review and comment
Thanks.

### CODE REVIEW

File: app/controllers/items/partial_undesignate.js [hunk 1]
```diff
@@ -13,57 +15,88 @@ export default Ember.Controller.extend({
   partialUndesignateBackLinkpath: Ember.computed.localStorage(),
 
   init() {
-    this.set('promptUserPopUp', false);
+    this.set("promptUserPopUp", false);
+  },
+
+  cannotDispatchItems(designation) {
```
Comment (namrataukirde) [inline app/controllers/items/partial_undesignate.js]: Is this criteria is same as dispatching from item details page? 

Just confirming :)

Comment (abhinav098) [reply app/controllers/items/partial_undesignate.js]: Yes

File: app/controllers/items/partial_undesignate.js [hunk 2]
```diff
@@ -13,57 +15,88 @@ export default Ember.Controller.extend({
...
+    return (
+      designation &&
```
Comment (Darshan-Chauhan) [inline app/controllers/items/partial_undesignate.js]: can we set this up in a const and return that const?? As it would be more readable

Comment (abhinav098) [reply app/controllers/items/partial_undesignate.js]: yeah, that can be done.

**Pull request merged**